src/components/editTestCase/qiCore/LeftPanel/ElementsTab/Demographics/DemographicsSection.tsx

```diff
@@ -141,6 +161,19 @@ const DemographicsSection = ({ canEdit }) => {
           <div className="demographics-container">
             <div className="demographics-row">
               <FormControl tw={"w-2/4"}>
+                <DateField
+                  label="Date of Birth"
+                  disabled={!canEdit}
+                  value={patient?.birthDate ? dayjs(patient.birthDate) : null}
+                  handleDateChange={(value: any) => {
+                    const date = new Date(value);
+                    const year = date.getFullYear();
+                    const month = String(date.getMonth() + 1).padStart(2, "0"); // Adding 1 because months are zero-indexed
```

Comment:
I think you can remove the commented out code

Reply:
Removed